Assets/Resources/defaults.ini.txt

```diff
@@ -134,7 +134,7 @@ WeaponAttackThreshold=0.03
 WeaponSensitivity=1.0
 MovementAcceleration=False
 ToggleSneak=False
-ClickToAttack = False
+AttackMode = 0
```

Comment:
Not a huge thing, but while I think "Click to Attack" wasn't a great setting name, I think "AttackMode" might sound a bit more vague. Maybe call it "WeaponSwingMode"?

Reply:
That makes sense code-wise. But should I also change the UI labels?

Reply:
I would say so. The name would make it more clear to the player of what kind of setting it is.